index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-cycle */
+import { bookListHandler, addSectionHandler, contactInfoHandler } from './modules/singlePage.js';
```

Comment:
- [ ] Let's try not to disable Linter checks in our codebase as they help us write better and more professional code. Disabling Linter checks defies the purpose of having it and hence does not help you write professional code. Kindly fix the Linter error 👍🏽